DXRVanilla/DeusEx/Classes/Player.uc

```diff
@@ -82,6 +82,7 @@ function DXRBase DXRFindModule(class<DXRBase> class)
 
 function PostIntro()
 {
+    strStartMap = class'DXRStartMap'.static.GetStartMap(self,flagbase.GetInt('Rando_starting_map'));
```

Comment:
if we can move all of this logic into MissionIntro then we can make this work for other mods